git_machete/cmd.py

```diff
@@ -44,31 +44,35 @@ def underline(s):
     return UNDERLINE + s + ENDC
 
 
+def star(f):  # tuple unpacking in lambdas
```

Comment:
Nice trick :+1:

Reply:
Yeah, that's what stackoverflow adviced to emulate tuple unpacking in lambdas under Python 3... since this feature is gone :(